### ISSUE TITLE
A Change in Tagcomplete?

### ISSUE THREAD

**Issue by Tijaani85** [DominikDoom/a1111-sd-webui-tagcomplete#130]
Hello, Ive noticed a certain change in how the tagcomplete   list works, particularly when trying to get it to display a list of embeddings. I use UmiAi and until about 2 weeks ago I could type  something like "<Char-F A..." And the tagcomplete would list all the embeddings from the CHAR-F folder from UmiAi that started with the letter A. 

Now when I type the same thing, there is no longer an autocomplete list shown. The tagcomplete still works but because if i simply type "<Char-F..." Itll show me a list of various embeddings from that folder(All filenames in said folder start with Char-F btw) but thats it, its always the same list  of embeddings and i cant get specific embeds by name with tagcomplete like i could in the past. Has something changed with the functionality of the tagcomplete display? Is there an option or a way to return it to how it used to work?

**Comment by DominikDoom** [comment]
I'm not aware of changing anything with the embed completions recently.
On the other hand, I'm more surprised that this would have worked at all for you at any point in time, since embedding completion never supported spaces in filenames. So <Char-F A... would have by any means just closed the list, since the A would be recognized as a new tag. It's also not doing any fancy folder based completion, just purely by filename.

Using spaces as break points is pretty much unavoidable since it wouldn't be possible otherwise to get embedding completion in the middle of a prompt without first adding a comma or other separator in front of what you're typing, which is pretty slow and annoying.

The best option really would be if the UMI devs didn't use spaces in their file names since that is almost expected to lead to issues with some systems sooner or later, but as a workaround I added the ability to use _ in place of spaces.

So TLDR: you can still do it now by typing <char-f_a... with underscores in place of spaces, it will still be replaced correctly.
